lib/signaling/v2/cancelableroomsignalingpromise.js

```diff
@@ -136,13 +137,12 @@ function createCancelableRoomSignalingPromise(token, wsServer, localParticipant,
     }
   });
 
-  cancelablePromise.catch(error => {
+  cancelablePromise.catch(() => {
     if (transport) {
       transport.disconnect();
       transport = null;
     }
     peerConnectionManager.close();
-    throw error;
```

Comment:
curious - why was this throw removed? Which case was it causing an issue?

Reply:
@makarandp0 This will never be caught because we are returning the CancelablePromise and not the Promise returned by `catch()`. So, anything thrown here will be displayed as an uncaught error in the JS console.